src/Search/Microsoft.Azure.Search/Customizations/Indexers/Models/IndexingParameters.cs

```diff
@@ -29,12 +25,11 @@ public partial class IndexingParameters
         /// <summary>
         /// Initializes a new instance of the IndexingParameters class.
         /// </summary>
-        public IndexingParameters(int? batchSize = default(int?), int? maxFailedItems = default(int?), int? maxFailedItemsPerBatch = default(int?), bool? base64EncodeKeys = default(bool?), IDictionary<string, object> configuration = default(IDictionary<string, object>))
+        public IndexingParameters(int? batchSize = default(int?), int? maxFailedItems = default(int?), int? maxFailedItemsPerBatch = default(int?), IDictionary<string, object> configuration = default(IDictionary<string, object>))
```

Comment:
No, because we are not committed to rolling this by hand forever. As soon as we extend AutoRest to understand deprecated parameters, this will go back to being machine-generated.